### ISSUE TITLE
Start keybase GUI Client minimized

### ISSUE THREAD

**Issue by steelbrain** [keybase/keybase-issues#2525]
I'm on OSX and it's annoying that it loads and is on top of everything else. It would be nice to have it minimized to tray on boot like any other application

**Comment by joehillen** [comment]
I would like this for Linux as well. Annoyingly there are not command-line flags for `run_keybase`

**Comment by asgeirn** [comment]
I'd love this on Windows as well.

**Comment by jesseschoff** [comment]
Please add this functionality. I don't want to have to close the Keybase app to the system tray in Windows every time my computer boots up.

**Comment by zQueal** [comment]
--minimized

would be a cool option. Just add it to the startup options by default.

**Comment by BostonEnginerd** [comment]
I agree. This would be a nice feature. The window popping up by default is a bit annoying.

**Comment by Attoy** [comment]
Totally agree on that, please add this feature!

**Comment by stefanotorresi** [comment]
reatedl: keybase/client#6031
